cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -18,15 +18,75 @@ option java_outer_classname = "CassandraFrameworkProtos";
 message CassandraFrameworkConfiguration {
     optional string frameworkId = 1;
     optional string frameworkName = 2;
-    optional string cassandraVersion = 3;
-    optional int32 numberOfNodes = 4;
-    optional double cpuCores = 5;
-    optional int64 memMb = 6;
-    optional int64 diskMb = 7;
     optional int64 healthCheckIntervalSeconds = 8;
-    optional int32 numberOfSeeds = 9;
     optional int64 bootstrapGraceTimeSeconds = 10;
     repeated PortMapping portMapping = 11;
+
+    // default configuration
+    required CassandraConfigRole defaultConfigRole = 6;
+    // additional configurations (currently unused)
+    repeated CassandraConfigRole configRoles = 7;
```

Comment:
If this isn't being used lets remove it. Protobuf is tolerant of fields being added later.